utils/makefile.python

```diff
@@ -97,13 +115,7 @@ quiet_cmd_pyinstall = INSTALL   $2
 
 # $2 path to folder with setup.py, this uses pip from pyenv (not OS!)
 quiet_cmd_pyenvinstall = PYENV     install $2
-      cmd_pyenvinstall = \
-	if ! cat $(PY_ENV)/requirements.sha256 2>/dev/null | sha256sum --check --status 2>/dev/null; then \
-		$(PY_ENV_BIN)/python -m pip $(PIP_VERBOSE) install -e $2$(PY_SETUP_EXTRAS) ;\
-		sha256sum requirements*.txt > $(PY_ENV)/requirements.sha256 ;\
-	else \
-		echo "PYENV     $2 already installed"; \
-	fi
+      cmd_pyenvinstall = $(PY_ENV_BIN)/python -m pip $(PIP_VERBOSE) install -e $2$(PY_SETUP_EXTRAS)
```

Comment:
Why remove the check about `requirements-dev.txt` ?

Without it, `make test` check each time the dependencies of `requirements-dev.txt`: everything since there is `-r requirements.txt`.

Reply:
> Why remove the check about requirements-dev.txt ?

The sha256sum of the `requirements*.txt` is done in the `cmd_virtualenv` function.

The function `cmd_pyenvinstall` does now try to `import searx,  pkg_resources` .. installation is only done if this import fails.